app/templates/components/head-layout.js

```diff
@@ -0,0 +1 @@
+export { default } from 'ember-cli-head/templates/components/head-layout';
```

Comment:
We should either colocate the template (e.g. move it from `addon/templates/components/head-layout.hbs` to `addon/components/head-layout.hbs`) or move to using `setComponentTemplate` in the class definition file (see other comment for exmaple of that).